cloudbuild/python-scripts/insert_dynamic_partitions.py

```diff
@@ -0,0 +1,146 @@
+"""
+Python script to dynamically partitions to a BigQuery partitioned table.
+"""
+import argparse
+import random
+from collections.abc import Sequence
+import datetime
+import threading
+import time
+from absl import app
+from utils import utils
+
+
+def wait():
+    print(
+        'Going to sleep, waiting for connector to read existing, Time:'
+        f' {datetime.datetime.now()}'
+    )
```

Comment:
Similar to comments in [#65](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/pull/65), let's utilize a logging framework instead of print to standard output.

Reply:
Fixed.